### DISCUSSION TITLE
This browser requires HTTPS to use the web vault Check the Vaultwarden wiki for details on how to enable it

### DISCUSSION THREAD

**Discussion by nitrup** [dani-garcia/vaultwarden#2609] [Q&A]
Hi team,

Since 5 day impossible to log on my bitwarden, error This browser requires HTTPS to use the web vault
Check the Vaultwarden wiki for details on how to enable it

Could you help me ?

Thanks

**Comment by BlackDex** [comment]
Well, that looks like you are not accessing the web-vault using https, just as the message tells you. Check that you are as accessing the site via https and try again.

**Comment by RealOrangeOne** [comment]
See also https://github.com/dani-garcia/vaultwarden/discussions/2274 for some more details, but basically you can't use Bitwarden without HTTPS.
